scripts/cloudfront_routing.js

```
@@ -0,0 +1,31 @@
+    var uri = request.uri;
+  
+    // Route mapping for static files and specific routes
+    var routeMap = {
+      "/": "/index.html",               // Root path -> serve the index.html file
+      "/about": "/about/index.html",    // '/about' -> '/about/index.html'
+      "/about/": "/about/index.html",   // '/about/' -> '/about/index.html'
+      "/en": "/en/index.html",          // '/en' -> '/en/index.html' (English homepage)
+      "/en/": "/en/index.html",         // '/en/' -> '/en/index.html'
+      "/swagger": "/swagger.html",      // '/swagger' -> '/swagger.html'
+      "/test": "/_next/static/media/desktop.0ec56f83.jpg" // Specific file for '/test'
+    };
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)